quickcache/quickcache_helper.py

```diff
@@ -121,6 +122,8 @@ def _serialize_for_key(self, value):
         elif isinstance(value, set):
             return 'S' + self._hash(
                 ','.join(sorted(map(self._serialize_for_key, value))))
+        elif isinstance(value, uuid.UUID):
```

Comment:
I think `isinstance` is a pretty expensive calculation right? at some point it may be useful to use `type(value)` once and have a dictionary of types -> parsers with `isinstance` as a fallback option.

Reply:
not sure on the perf of it, was just following the convention, but your suggestion makes sense to me

Reply:
yeah. not meant for you to do with this PR, but just a thought for me / @dannyroberts ( or you if you're feeling up to it)